### PR TITLE
Update steering committee membership in sigs.yaml

### PR DESCRIPTION
**Pull request by cblecker**
/cc @nikhita @parispittman @dims @spiffxp @derekwaynecarr @timothysc

### CODE REVIEW

File: committee-steering/README.md [hunk 1]
```diff
@@ -18,16 +18,11 @@ The [charter](https://git.k8s.io/steering/charter.md) defines the scope and gove
 
 ## Members
 
-* Brian Grant (**[@bgrant0607](https://github.com/bgrant0607)**), Google
-* Brendan Burns (**[@brendandburns](https://github.com/brendandburns)**), Microsoft
+* Christoph Blecker (**[@cblecker](https://github.com/cblecker)**), Red Hat
```
Comment (timothysc) [inline committee-steering/README.md]: FWIW, I think it's worth calling out Emeritus members as well.

Comment (cblecker) [reply committee-steering/README.md]: We hadn't previously done this in sigs.yaml for steering before (Tim and Quinton aren't there), but had in git.k8s.io/steering/README.md. Happy to add them here too if we'd like

Comment (timothysc) [reply committee-steering/README.md]: The readme is sufficient, I've long since lost track of all the redundant places we've scrawled things down...

**Pull request merged**